packages/components/src/checkbox/docs/Checkbox.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import { Checkbox } from "../src/Checkbox.tsx";
+import type { Meta, StoryObj } from "@storybook/react";
+import { Text } from "../../Text/index.ts";
+import { Stack } from "../../layout/index.ts";
+import { PlusIcon } from "@hopper-ui/icons";
+import { IconList } from "../../IconList/index.ts";
+
+/**
+ * The Checkbox component is used to select or deselect an option. Indeterminate state is also supported.
+ *
+ * [View repository](https://github.com/gsoft-inc/wl-hopper/tree/main/packages/components/src/Checkbox/src)
+ * -
+ * [View package](https://www.npmjs.com/package/@hopper-ui/components)
+ * -
+ * View storybook TODO
+ */
+const meta = {
+    title: "Docs/Checkbox/Checkbox",
+    tags: ["autodocs"],
+    parameters: {
+        // Disables Chromatic's snapshotting on documentation stories
+        chromatic: { disableSnapshot: true }
+    },
+    component: Checkbox
+} satisfies Meta<typeof Checkbox>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+/**
+ * A default checkbox.
+ */
+export const Default: Story = {
+    args: {
+        children: "Option 1"
+    }
+};
+
+/**
+ * A checkbox can be rendered without a label. In this case, the `aria-label` attribute should be used to provide a label to the checkbox.
+ */
+export const NoLabel: Story = {
+    render: props => (
+        <Checkbox aria-label="Option 1" {...props} />
+    )
+};
+
+/**
+ * A checkbox can be rendered in a checked state.
+ */
+export const Checked: Story = {
+    args: {
+        ...Default.args,
+        defaultSelected: true
+    }
+};
+
+/**
+ * A checkbox can be rendered in an indeterminate state.
+ */
+export const Indeterminate: Story = {
```

Comment:
the inderteminate sample should be checked by default in the doc.

<img width="477" alt="image" src="https://github.com/gsoft-inc/wl-hopper/assets/361632/258b8ca5-414a-4b0c-b1a1-b8100a4babb2">